Kernel/Signature.hpp

```diff
@@ -157,6 +142,7 @@ class Signature
     unsigned _letBound : 1;
     /** proxy type */
     Proxy _prox;
+    int _dbIndex;
```

Comment:
Maybe, could we spell out "DeBruijn" in full? 

For a second, I started thinking about databases ...